CMakeLists.txt

```diff
@@ -0,0 +1,336 @@
+# minimum required vertion of cmake
+cmake_minimum_required (VERSION 3.2)
+
+# project name
+project (raqm)
+
+##############################3
+
+#enable compiler warning for GNU C
+if(CMAKE_COMPILER_IS_GNUCC)
+add_definitions (-Wall  -Wno-portability)
+#enable C99 mode
+add_definitions(-std=gnu99)
+endif()
+
+#enable compiler warning for MSVC
+if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+  set(CMAKE_C_FLAGS "/W4 /WX /wd4068")
+endif()
+
+#not working yet.. check
+#if(WIN32 AND NOT(MSYS OR CYGWIN OR MINGW) )
+#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /W4 /wd4068")
+#endif()
+
+
+
+#set the values of raqam.pc file
+set (prefix /usr/local)
+set (exec_prefix ................................)
+set (libdir /lib)
+set (includedir /include)
+set (PACKAGE_NAME "Raqm")
+set (PACKAGE_VERSION 0.2.0)
+set (AX_PACKAGE_REQUIRES "freetype2 >= 12.0.6")  
+set (AX_PACKAGE_REQUIRES_PRIVATE "harfbuzz fribidi")

```

Comment:
Same here, this duplicates information found elsewhere, so making a new release I have to rememver all the places to update the version number, or when updating the required versions of the packages.